.github/workflows/tests-unit-php.yml

```diff
@@ -45,19 +45,16 @@ jobs:
     continue-on-error: ${{ matrix.experimental == true }}
     strategy:
       matrix:
-        php: ['7.0', '5.6']
+        php: ['8.0', '7.0']
         wp: ['latest']
         coverage: [false]
         include:
           - php: '7.4'
             wp: 'latest'
             coverage: true
 
-          - php: '5.6'
-            wp: '5.3'
-
-          - php: '8.0'
```

Comment:
Why is the `8.0` removed here, too?

Reply:
It was just moved to line above ` php: ['8.0', '7.0']`

Reply:
<img width="355" alt="Screenshot 2021-06-18 at 11 38 40" src="https://user-images.githubusercontent.com/237508/122548836-c1e24480-d029-11eb-9daa-efd9b8cac8e5.png">
You can see it still test again 8.0